scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -536,3 +540,68 @@ private[scalding] case class FlowStepStrategies[A]() extends Semigroup[FlowStepS
       }
     }
 }
+
+/*                                                                             
+ * Run a pre-existing MapReduce job with Scalding.
+ * Note that it is up to the user to properly configure the job.                                      
+ */
+class MapReduceJob(args: Args) extends Job(args) {
+  val conf: Configuration = mode match {
+    case h: HadoopMode => h.jobConf
+    case _ => new Configuration
+  }
+  protected val job = new JobConf(conf, classOf[MapReduceJob])
```

Comment:
can we call this jobConf or something? Also, can we put a type or make it private?